### PR TITLE
Handle alerts without description

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [na] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

This PR fixes an error in the email message rendering that prevents email notifications from being sent if they are about alerts with a null description text.

### CODE REVIEW

File: src/main/java/org/opentripplanner/middleware/otp/response/LocalizedAlert.java [hunk 1]
```diff
@@ -46,31 +57,19 @@ public String getAlertDescriptionForHtml() {
...
         LocalizedAlert ao = (LocalizedAlert) o;
-        if (alertDescriptionText == null) {
-            if (ao.alertDescriptionText != null) {
-                return false;
-            }
-        } else {
-            if (!alertDescriptionText.equals(ao.alertDescriptionText)) {
-                return false;
-            }
+        if (!getAlertDescriptionText().equals(ao.getAlertDescriptionText())) {
```
Comment (br648) [inline src/main/java/org/opentripplanner/middleware/otp/response/LocalizedAlert.java]: I think these two if statement can be combined.

**Review by br648 — APPROVED** [review]
Just one minor comment. Apart from that all good.

File: src/main/java/org/opentripplanner/middleware/otp/response/LocalizedAlert.java (hunk 1 above)
Comment (binh-dam-ibigroup) [reply src/main/java/org/opentripplanner/middleware/otp/response/LocalizedAlert.java]: Updated in https://github.com/ibi-group/otp-middleware/pull/215/commits/4c14f95ab15888c47bf285cd5a9f688d49f3f94b.

**Review by amy-corson-ibigroup — APPROVED** [review]
Makes sense to me! Thanks!

**Pull request merged**